unittest/resources/test_user.py

```diff
@@ -13,11 +13,12 @@ def mySetup(self):
     def myTeardown(self):
         pass
 
+
 class TestUser_get(TestUser):
     """Test cases for the get API endpoint.
     """
-    def test_something(self):
-        """Sample test.
+    def test_get_success(self):
+        """Test user is successfully returned.
```

Comment:
To be consistent across your other unit tests, lets make the doc string `"Verify user is successfully returned."`